app/assets/javascripts/admin/views/broken-links-report.js

```diff
@@ -0,0 +1,76 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function BrokenLinksReport (module) {
+    this.module = module
+  }
+
+  BrokenLinksReport.prototype.init = function () {
+    if (!fetch) return
+    var button = this.module.querySelector('button')
+    var authenticityTokenField = this.module.querySelector('[name="authenticity_token"]')
+    if (button && authenticityTokenField) this.setupEventListener(button, authenticityTokenField.value)
+    var refreshLink = this.module.querySelector('.js-broken-links-refresh')
+    if (refreshLink) {
+      this.setupPolling(refreshLink.href)
```

Comment:
This passing in the href to start polling
```
      this.setupPolling(refreshLink.href)
```

but the function has two expected variables:

```
startPolling = function (wait, polledFunction)
```

where is this set?

Reply:
`setupPolling` and `startPolling` were two different functions.
I've restructured to `setupPolling` and `poll` to simplify things.